packages/docs/src/examples/va-modal/HidingOverlay.vue

```diff
@@ -1,20 +1,28 @@
 <template>
   <p>
-    <va-button class="mr-4" @click="showModal = !showModal">
-      Show modal (without the overlay)
```

Comment:
This one could probably be split into generic/non-generic (in brackets) parts.

Reply:
the sentence separated and `Show modal` moved to generic translations